src/prefect/flows.py

```diff
@@ -1925,8 +1925,12 @@ def load_flow_argument_from_entrypoint(
                     literal_arg_value = ast.get_source_segment(
                         source_code, keyword.value
                     )
+                    cleaned_value = (
+                        literal_arg_value.replace("\n", "") if literal_arg_value else ""
+                    )
```

Comment:
Maybe another way to do this is to eval the expression wrapped in parenthesis?  Like `eval(f"({literal_arg_value})")` so the line breaking and spacing don't matter?

Reply:
I can try it!

Reply:
😞 

```
INFO     prefect.flows:flows.py:1930 Failed to parse @flow argument: `name=f"{flow_base_name}-"
        f"{version}"` due to the following error. Ignoring and falling back to default behavior.
Traceback (most recent call last):
  File "/Users/chrispickett/src/prefect/src/prefect/flows.py", line 1928, in load_flow_argument_from_entrypoint
    evaluated_value = eval(f"{(literal_arg_value)}", namespace)  # type: ignore
  File "<string>", line 2
    f"{version}"
IndentationError: unexpected indent```

Reply:
womp womp